### PR TITLE
feat: add cc docs homepage icons

### PR DESCRIPTION
**Pull request by christian-dinh**
### Overview

adding the icons we'll need for the cc docs homepage section titles

### PR Checklist

- [x] Related to designs: [ccdocs figma](https://www.figma.com/file/nkBzG9nJW36t8kDOBcvnyU/Codecademy-Docs---MVP?node-id=592%3A3482)
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.6.0-alpha.0d5fba.0<br/>@codecademy/gamut-kit@0.4.61-alpha.0d5fba.0<br/>@codecademy/gamut-labs@19.15.4-alpha.0d5fba.0<br/>@codecademy/gamut@34.13.3-alpha.0d5fba.0<br/>@codecademy/styleguide@41.12.4-alpha.0d5fba.0<br/></pre>

**Review by MattSchiller — COMMENTED** [review]
It doesn't look like these svgs have been run through our `svgo`?

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60df6a53602b684790efde79--gamut-preview.netlify.app>https://60df6a53602b684790efde79--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60df6a53602b684790efde79>Deploy Logs</a></p>

**Comment by christian-dinh** [comment]
I think that happens at build time? I did [all the steps](https://gamut.codecademy.com/storybook/?path=/docs/atoms-icons-regular--editable-icon#adding-an-icon) mentioned in gamut and all the output was in ignored files. I think this explains (from the same page as above):

>All of our Icons are not minified by default, and optimized by SVGO before build time. We like to preserve the source URL as well as any relevent comments or metadata within these SVGs.

**Comment by MattSchiller** [comment]
> I think that happens at build time? I did [all the steps](https://gamut.codecademy.com/storybook/?path=/docs/atoms-icons-regular--editable-icon#adding-an-icon) mentioned in gamut and all the output was in ignored files. I think this explains (from the same page as above):
> 
> > All of our Icons are not minified by default, and optimized by SVGO before build time. We like to preserve the source URL as well as any relevent comments or metadata within these SVGs.

Ah, gotcha. Thanks for TIL'ing me!

File: packages/gamut-icons/src/svg/regular/file-code-add-1-icon.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/file-code-add-1-icon.svg:3]:
```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

these ids can have some weird side effects

File: packages/gamut-icons/src/svg/regular/multiple-actions-view-icon.svg [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Regular" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (dreamwasp) [inline packages/gamut-icons/src/svg/regular/multiple-actions-view-icon.svg:3]:
```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

**Review by dreamwasp — APPROVED** [review]
just remove those ids and everything looks ✨

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.6.0-alpha.37f1dd.0<br/>@codecademy/gamut-kit@0.4.62-alpha.37f1dd.0<br/>@codecademy/gamut-labs@19.15.5-alpha.37f1dd.0<br/>@codecademy/gamut@34.14.1-alpha.37f1dd.0<br/>@codecademy/styleguide@41.13.1-alpha.37f1dd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ec604af3ff0950023a6546--gamut-preview.netlify.app>https://60ec604af3ff0950023a6546--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ec604af3ff0950023a6546>Deploy Logs</a></p>

**Pull request merged**